JASP-Engine/JASP/R/ttestindependentsamples.R

```diff
@@ -699,13 +699,27 @@ TTestIndependentSamples <- function(dataset = NULL, options, perform = "run",
 				 yend = yend), inherit.aes = FALSE, size = 1),
 				 ggplot2::scale_y_continuous(breaks = c(min(b), max(b))))
 		}
-
+		
 		for (var in .indices(variables)) {
-			descriptivesPlot <- list("title" = variables[var])
-			descriptivesPlot[["width"]] <- options$plotWidth
-			descriptivesPlot[["height"]] <- options$plotHeight
-			descriptivesPlot[["custom"]] <- list(width = "plotWidth", height = "plotHeight")
-
+		    
```

Comment:
Please change var to variable or variableName, there's also a var function in stats::var